janitor/tasks/labware_location/sql_queries/get_labware_locations.sql

```diff
@@ -31,4 +31,4 @@ WHERE
 	audits_b.updated_at IS NULL
 	AND audits.updated_at >= %(latest_timestamp)s;
```

Comment:
I think the `;` at the end of this line means the `ORDER BY` below doesn't get applied.

Reply:
Maybe this was the true cause of the error messages seen.

Reply:
Yes you are correct, that must be from before we added the `ORDER BY`, so I've just removed it. Thank you for pointing it out!